include/tesseract/baseapi.h

```diff
@@ -342,11 +342,7 @@ class TESS_API TessBaseAPI {
    * Note that Tesseract takes ownership of the Thresholder and will
    * delete it when it it is replaced or the API is destructed.
    */
-  void SetThresholder(ImageThresholder *thresholder) {
-    delete thresholder_;
-    thresholder_ = thresholder;
-    ClearResults();
-  }
+  void SetThresholder(ImageThresholder *thresholder);
```

Comment:
This can't be a public API anymore, since user does not know how to inherit from ImageThresholder.

Reply:
So should be removed in this case.

Reply:
Right, so that should be removed, too, if it is not used anywhere.

Reply:
I added a commit now which removes `TessBaseAPI::SetThresholder`, too.